config/development.json

```
@@ -76,6 +76,7 @@
 		"help": true,
 		"help/gpt-response": true,
 		"home/layout-dev": true,
+		"hosting-server-settings-enhancements": false,
```

```suggestion
		"hosting-server-settings-enhancements": true,
```
Nit, but it would be nice to default to true in dev environments.

---

I explained my point in the description:

```
I decided to turn off the feature flag by default, to avoid confusion for other code wranglers. So we (letero) will develop with own local feature flag turned on -- create .env file in the root directory of Calypso with the next content ACTIVE_FEATURE_FLAGS='hosting-server-settings-enhancements'. And don't forget to restart local Calypso dev-server to apply this feature flag.
```

WDYT about it?
I don't have strong objection against using `development.json` as `true`

---

Right. I would probably still change the default to true. I don't think we should land this PR until the buttons are functional, and if they are, I don't think other code wranglers would be very confused by this.